sdk-extensions/autoconfigure/src/main/java/io/opentelemetry/sdk/autoconfigure/OtlpConfigUtil.java

```diff
@@ -73,10 +73,9 @@ static void configureOtlpExporterBuilder(
       setEndpoint.accept(endpoint.toString());
     }
 
-    Map<String, String> headers = config.getMap("otel.exporter.otlp." + dataType + ".headers");
-    if (headers.isEmpty()) {
-      headers = config.getMap("otel.exporter.otlp.headers");
-    }
+    Map<String, String> fallbackHeaders = config.getMap("otel.exporter.otlp.headers");
```

Comment:
Let's revert these two lines for similar reasons to above. When reading two properties in priority order, I think it is a lot easier to read when the code also goes in that order.